packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -41,7 +39,7 @@ export interface UseModalAttributesOptions {
 export const useModalAttributes = (
   options: UseModalAttributesOptions = {},
 ): { modalAttributes: TabsterTypes.TabsterDOMAttribute; triggerAttributes: TabsterTypes.TabsterDOMAttribute } => {
-  const { trapFocus, alwaysFocusable, legacyTrapFocus } = options;
+  const { trapFocus, alwaysFocusable, legacyTrapFocus = true } = options;
```

Comment:
Is it a breaking change in the behavior?

Reply:
I marked it as a minor version change -- my thought behind it not being breaking is that both behaviors are modal focus traps that prevent focus from reaching the background page -- it's just a change of opinion in which approach should be our default / the standard.

Reply:
LMK if you disagree -- we could alternatively just change the docs & DatePicker, and have it be an internal practice to set `legacyTrapFocus` to true

Reply:
@smhigley as it's a public API, IMO we should not change defaults in patches/minors. Let's go with this option:

> we could alternatively just change the docs & DatePicker, and have it be an internal practice to set legacyTrapFocus to true

Reply:
@layershifter done!